src/datasources/targeted-messaging/entities/__tests__/outreach-file.builder.ts

```diff
@@ -0,0 +1,23 @@
+import { Builder, type IBuilder } from '@/__tests__/builder';
+import type { OutreachFile } from '@/datasources/targeted-messaging/entities/outreach-file.entity';
+import { DB_MAX_SAFE_INTEGER } from '@/domain/common/constants';
+import { faker } from '@faker-js/faker/.';
+
+export function outreachFileBuilder(): IBuilder<OutreachFile> {
+  return new Builder<OutreachFile>()
+    .with(
+      'campaign_id',
+      faker.number.int({ min: 1, max: DB_MAX_SAFE_INTEGER - 1 }),
+    )
+    .with('campaign_name', faker.string.alphanumeric())
+    .with('team_name', faker.string.alphanumeric())
+    .with('start_date', faker.date.recent())
+    .with('end_date', faker.date.recent())
+    .with(
+      'safe_addresses',
+      Array.from(
+        { length: faker.number.int({ min: 10, max: 50 }) },
+        () => faker.finance.ethereumAddress() as `0x${string}`, // not checksummed
```

Comment:
Nit: I'd suggest we return a checksummed address here and explicitly set non-checksummed ones where tested for clarity.

Reply:
Changed in [3b047f9](https://github.com/safe-global/safe-client-gateway/pull/2033/commits/3b047f9a79b4ab6aaec40cbdb3e34482c58b0c10), thanks!